lib/ui/painting/image_decoder_unittests.cc

```diff
@@ -533,61 +547,30 @@ TEST(ImageDecoderTest, VerifySimpleDecoding) {
   ASSERT_TRUE(image != nullptr);
   ASSERT_EQ(SkISize::Make(600, 200), image->dimensions());
 
-  ASSERT_EQ(ImageFromCompressedData(data, 6, 2, ImageUpscalingMode::kNotAllowed,
-                                    fml::tracing::TraceFlow(""))
-                ->dimensions(),
-            SkISize::Make(6, 2));
-}
-
-TEST(ImageDecoderTest, VerifySimpleDecodingNoUpscaling) {
-  auto data = OpenFixtureAsSkData("Horizontal.jpg");
-  auto image = SkImage::MakeFromEncoded(data);
-  ASSERT_TRUE(image != nullptr);
-  ASSERT_EQ(SkISize::Make(600, 200), image->dimensions());
+  auto codec = SkCodec::MakeFromData(data);
+  ASSERT_TRUE(codec);
+  auto descriptor =
+      fml::MakeRefCounted<ImageDescriptor>(std::move(data), std::move(codec));
 
   ASSERT_EQ(
-      ImageFromCompressedData(data, 900, 300, ImageUpscalingMode::kNotAllowed,
-                              fml::tracing::TraceFlow(""))
+      ImageFromCompressedData(descriptor, 6, 2, fml::tracing::TraceFlow(""))
           ->dimensions(),
-      SkISize::Make(600, 200));
-}
-
-TEST(ImageDecoderTest, VerifySimpleDecodingNoUpscalingOneDimension) {
```

Comment:
This functionality has been moved over to the dart side and is tested in resize_image_test.dart